setup.sh

```diff
@@ -83,25 +83,13 @@ clean() {
         fi
 }
 
-import_shared_dev_route53 () {
-# Imports into terraform the shared development route 53 zone that had not been set up
-# using Terraform
-
-        # check if the resource exists in the state file
-        SHARED_DEV_SUBDOMAIN=`aws-vault exec ${PROFILE_NAME} -- $TERRAFORMPATH state list $SHARED_DEV_SUBDOMAIN_RESOURCE`
-        if [ "$SHARED_DEV_SUBDOMAIN" = "$SHARED_DEV_SUBDOMAIN_RESOURCE" ] ; then
-                echo "$SHARED_DEV_SUBDOMAIN already imported"
-        else
-                echo "import $SHARED_DEV_SUBDOMAIN_RESOURCE"
-                aws-vault exec ${PROFILE_NAME} -- $TERRAFORMPATH import $SHARED_DEV_SUBDOMAIN_RESOURCE $SHARED_DEV_DNS_ZONE
```

Comment:
We can probably also remove the `SHARED_DEV_DNS_ZONE` defined at the top of the file.